views/layouts/default.handlebars

```diff
@@ -3,13 +3,29 @@
 		<head>
 		<meta charset="UTF-8">
 			<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.4/angular.js"></script>
-			<script src="js/build/main.js" type="text/javascript"></script>
 			<title>{{ seoPageTitle }}</title>
     		<link rel="stylesheet" href="css/build/main.css">
 	</head>
-	<body ng-app="ProRubric">
-		<div ng-controller="mainController">
-		{{{ body }}}
-        </div>
+	<body>
+    <body ng-app="ProRubric">
+    <div ng-controller="mainController">
+
+		<nav>
+			<a href="#"><img src="http://i.imgur.com/awDgrl8.png" id="logo"></a>
+			<a href=""><img src="http://i.imgur.com/O2PDYmn.png">Search</a>
+			<a href="" ><img src="#">Degree</a>
+			<a href="" ><img src="#">Class</a>
+		</nav>
+
+		<div id="elBody">
```

Comment:
elBody is not descriptive enough of a class. elemBody is however. Yes I know this is picky, but there is a best practice associated to my comment. A three letter prefix is optimal as it is a standard that can be used more widely than 2.  Example: ary, obj, int, str, fnc, num. 

Remember the goal of a variable, class or function name is to imply uncontroveratble meaning. When in doubt, go longer. elBody is just a bit small. Is it The Body, or element body... either way it is not the element body, if anything it is a virtual body.

Please reconsider and refactor. (and then proceed to hate me)